# generated by patch-package 6.5.0 on 2022-03-31 20:04:44
#
# command:
#   npx patch-package @emotion/utils --exclude nothing
#
# declared package:
#   @emotion/utils: 1.1.0
#
diff --git a/node_modules/@emotion/utils/package.json b/node_modules/@emotion/utils/package.json
index d661529..e5738e8 100644
--- a/node_modules/@emotion/utils/package.json
+++ b/node_modules/@emotion/utils/package.json
@@ -4,10 +4,6 @@
   "description": "internal utils for emotion",
   "main": "dist/emotion-utils.cjs.js",
   "module": "dist/emotion-utils.esm.js",
-  "browser": {
-    "./dist/emotion-utils.cjs.js": "./dist/emotion-utils.browser.cjs.js",
-    "./dist/emotion-utils.esm.js": "./dist/emotion-utils.browser.esm.js"
-  },
   "types": "types/index.d.ts",
   "license": "MIT",
   "scripts": {
